support/oai_pmh.erl

```diff
@@ -21,37 +31,42 @@ import(Endpoint, Context) ->
 
 %% @doc Send a notification for each record retrieved from an OAI-PMH endpoint
 import(Endpoint, UrlParams, Context) ->
-    import([], Endpoint, UrlParams, Context).
+    {Records, Token} = list_records(Endpoint, UrlParams),
+    import(#import{records = Records,
+                   resumption_token = Token,
+                   endpoint = Endpoint,
+                   context = Context,
+                   url_params = UrlParams}).
 
-import([], Endpoint, UrlParams, Context) ->
-    {Records, ResumptionToken} = list_records(Endpoint, UrlParams),
-    import(Records, Endpoint, UrlParams, ResumptionToken, Context).
+%%-----------------------------------------------------------------------------
+%% Internal functions
+%%-----------------------------------------------------------------------------
```

Comment:
Out of curiosity: why write this? All non-exported functions should simply be considered internal. (If you need to export a function to make it testable, that’s probably due to bad design.)